src/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php

```diff
@@ -14,22 +14,12 @@
 use Symfony\Component\Console\Helper\ProgressBar;
 use Symfony\Component\Console\Helper\Helper;
 use Symfony\Component\Console\Output\StreamOutput;
-use Symfony\Component\Console\Tests;
-
-require_once __DIR__.'/../ClockMock.php';
 
+/**
+ * @group clockMock
```

Comment:
Not a big fan of the group name. Also, wouldn't it be better for it to be an annotation instead of a group?